documentation/Demo-Server-Guide.md

```diff
@@ -184,9 +185,9 @@ sudo docker exec -ti mflow-on-8000_app_1 cat /var/log/application_manager/graphi
 
 ```bash
 # to stop
-sudo docker-compose --project-name 'mflow-on-8000' stop
+sudo docker-compose -E --project-name 'mflow-on-8000' stop
 # to remove (when new version is out)
-sudo docker-compose --project-name 'mflow-on-8000' down
+sudo docker-compose -E --project-name 'mflow-on-8000' down
```

Comment:
it doesn't need the PORT_APP and PORT_DASH passed as well?

Reply:
Yeah, you're right. Stand by...

Reply:
Done 5d8ddac1